app/views/admin/notes/_show.html.slim

```diff
@@ -0,0 +1,23 @@
+header.small
+  .post-info
+    .item
+      i.fa.fa-calendar-o>
+      = l note.created_at.to_date
+    .item
+      a href=admin_person_path(note.author)
+        i.fa.fa-user>
+        = note.author.name
+    - if policy(note).edit?
+      .item
+        a.edit-action href="#"
+          i.fa.fa-pencil>
+          = t(:edit)
+    - if policy(note).destroy?
+      .item
+        => link_to admin_note_path(note), method: :delete, class: 'delete-action',
+            data: { confirm: t('confirm_deletion', thing: t('common.note')) },
```

Comment:
This kind of I18n interpolation is not a good idea because in some languages (e.g. French) the word before `thing` depends on `thing`. In French it could be 'ce' or 'cette'. I know this is used in a couple other places but in most places we have separate strings. Can you please refactor this and the other two and get rid of that I18n key?